src/concepts/metric_hub.md

```diff
@@ -316,6 +351,32 @@ Statistics on top of these metrics need to be defined in the [`looker/` folder i
 
 To get more statistics added, please reach out on the [#data-help](https://mozilla.slack.com/archives/C4D5ZA91B) Slack channel.
 
+To filter and segment metrics in Looker, data sources that expose fields as dimensions can be configured in metric-hub. These base field data sources need to be joined with the metric data sources. Wildcard characters can be used to apply these joins to multiple data sources:
+
+```toml
+[data_sources.looker_base_fields]
+select_expression = """
+    SELECT
+        submission_date,
+        client_id,
+        os,
+        country,
+        channel
+    FROM
+        mozdata.telemetry.clients_daily
+"""
+columns_as_dimensions = true  # expose the selected fields as dimensions in Looker
+
+# Join `looker_base_fields` on to all the data sources in the file
```

Comment:
So the wildcard scope is limited to the file, as opposed to everything that would get resolved for the current app?

Reply:
It's limited to the current app actually. So `[data_sources.'*']` would also apply the configuration that come from other config files related to the app that are in scope.

Reply:
Ok, makes sense, that's what I expected. Thanks for clarifying! Maybe we can edit the comment to something like this?
```suggestion
# Join `looker_base_fields` on to all the data sources that are in scope for the current file (i.e., data sources for the current application)
```